src/Tasks.UnitTests/Exec_Tests.cs

```diff
@@ -1033,6 +1033,19 @@ public void EndToEndMultilineExec_EscapeSpecialCharacters()
                 }
             }
         }
+
+        [Fact]
+        public void ConsoleOutputDoesNotTrimLeadingWhitespace()
+        {
+            Exec exec = PrepareExec("type .\\Exec_Tests.Attachments\\leading-whitespace.txt");
```

Comment:
In order to keep the amount of test files limited, I suggest using `TestEnvironment` to create this one-line file during the test execution, as it was done in `EndToEndMultilineExec_EscapeSpecialCharacters` test above. It would be also help with having this line in one place instead of two.

Reply:
Thanks for suggestion! Done.